src/intelli/gui/property_item/finddirectchildnodewidget.h

```diff
@@ -0,0 +1,81 @@
+/* GTlab - Gas Turbine laboratory
+ * Source File:
+ * copyright 2009-2023 by DLR
+ *
+ *  Created on: 07.06.2024
+ *  Author: Jens Schmeink (AT-TWK)
+ *  Tel.: +49 2203 601 2191
+ */
```

Comment:
```suggestion
```suggestion
/*
 * GTlab IntelliGraph
 *
 *  SPDX-License-Identifier: BSD-3-Clause
 *  SPDX-FileCopyrightText: 2024 German Aerospace Center
 *
 *  Author: Jens Schmeink <jens.schmeink@dlr.de>
 */
```
```